docs/server/source/cloud-deployment-templates/ca-installation.rst

```diff
@@ -1,110 +1,108 @@
 Set Up a Self-Signed Certificate Authority
 ==========================================
 
-This document enumerates the steps we recommend to set up a self-hosted or a
-self-signed CA.
+This page enumerates the steps *we* use to set up a self-signed certificate authority (CA).
+Your organization might do it differently.
 
-There are many ways to get this done and you are free to follow the methods
-that you are most comfortable with.
-
-We currently use
-`easy-rsa version 3
+We use
+`Easy-RSA version 3
 <https://community.openvpn.net/openvpn/wiki/EasyRSA3-OpenVPN-Howto>`_, a 
-wrapper over complex ``openssl`` commands and helps us achieve this goal
-easily. It is available
-`here <https://github.com/OpenVPN/easy-rsa/releases>`_ and licensed under GPLv2.
+wrapper over complex ``openssl`` commands.
+`Easy-RSA is available on GitHub <https://github.com/OpenVPN/easy-rsa/releases>`_ and licensed under GPLv2.
 
 
-Step 1: Install Dependencies for CA Setup
------------------------------------------
+Step 1: Install Easy-RSA Dependencies
+-------------------------------------
 
-The only dependency for easy-rsa v3 is the presence of the ``openssl``
+The only dependency for Easy-RSA v3 is the presence of the ``openssl``
 command, which is available from the ``openssl`` package on Ubuntu and other
-Debian-based operating systems.
+Debian-based operating systems, i.e. you can install it using:
 
-``sudo apt-get install openssl`` should be enough in most cases.
+.. code:: bash
 
+   sudo apt-get update
 
-Step 2: Setup the easyrsa Package for CA Setup
-----------------------------------------------
+   sudo apt-get install openssl
 
-Download and extract the latest package.
 
-    .. code:: bash
-        
-        mkdir federation-ca && cd federation-ca
+Step 2: Install Easy-RSA
+------------------------
 
-        wget https://github.com/OpenVPN/easy-rsa/archive/3.0.1.tar.gz
+Create a directory named ``federation-ca/easy-rsa-3.0.1``,
+then download and extract the latest Easy-RSA package into it:
 
-        tar xzvf 3.0.1.tar.gz
+.. code:: bash
+        
+   mkdir federation-ca && cd federation-ca
 
+   wget https://github.com/OpenVPN/easy-rsa/archive/3.0.1.tar.gz
 
-This should give you a ``federation-ca/easy-rsa-3.0.1`` directory.
+   tar xzvf 3.0.1.tar.gz
 
 
 Step 3: Customize the CA Configuration
 --------------------------------------
 
-We customize the CA setup by creating a ``vars`` file based on the existing
-``vars.example`` file bundled in the package.
-
-The values for the ``Distinguished Name`` given below (country, province,
-city, org, email) are references and can be changed as per your
-deployment. Refer to the ``vars.example`` file for detailed variable descriptions.
+We customize the CA configuration by creating a ``vars`` file based
+on the existing ``vars.example`` file.
+The country, province, city, org and email values should be changed 
+to the values for your organization.
```

Comment:
Okay I added a note about those things being part of the DN in commit https://github.com/bigchaindb/bigchaindb/pull/1456/commits/424839a96b84ecc4da81474a8f264db096c89c8e